controllers/tasks.js

```diff
@@ -247,6 +247,23 @@ const overdueTasks = async (req, res) => {
   }
 };
 
+const overdueTasksAll = async (req, res) => {
+  try {
+    const overdueTasks = await tasks.overdueTasksAll();
+    const overdueTasksFiltered = overdueTasks.filter(
+      (task) =>
+        task.status !== MERGED || task.status !== COMPLETED || task.status !== RELEASED || task.status !== VERIFIED
+    );
+    return res.json({
+      message: "Overdue Tasks returned successfully!",
+      overdueTasks: overdueTasksFiltered,
+    });
+  } catch (err) {
+    logger.error(`Error while fetching overdue tasks : ${err}`);
+    return res.boom.badImplementation("An internal server error occured");
```

Comment:
[optional] Good to have method specific error message.

Reply:
Elaborate...

Reply:
Map actual error message to something more readable, instead of sending 
"An internal server error occured". One size shouldn't be fitting for all.

Reply:
Will the user be better equipped to do changes or fix the issues if they got to know the exact issue or is the developer better equipped for the real issue which they will get from the logger?

Reply:
Developer is always better equipped with `logger`, but `api` user is developer itself.
Anyways `logger` wins!!